src/main/java/com/revaturelabs/ask/question/QuestionController.java

```diff
@@ -164,6 +164,25 @@ public Question createOrUpdate(@RequestBody Question question, @PathVariable int
           "Error during update/creation of question", e);
     }
   }
+  
+  /**
+   * Gets questions that experts have responded to
+   * 
+   * @param expertId is the ID of the desired expert
+   */
+  @GetMapping("/{expertId}/response")
```

Comment:
This mapping means that hitting the endpoint "/questions/1/response" gets a list of questions produced by expert id 1.  That's not good.  users/1/answeredQuestions or similar would be better.  You could potentially use a query string param

Reply:
/questions?answeredBy=expertId maybe?